src/med_bench/estimation/mediation_g_computation.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+
+from med_bench.estimation.base import Estimator
+from med_bench.utils.decorators import fitted
+from med_bench.utils.utils import is_array_integer
+
+
+class GComputation(Estimator):
+    """GComputation estimation method class
+    """
+
+    def __init__(self, **kwargs):
+        """Initalization of the GComputation estimation method class
+        """
+        super().__init__(**kwargs)
+
+    def fit(self, t, m, x, y):
+        """Fits nuisance parameters to data
+
+        """
+        t, m, x, y = self._resize(t, m, x, y)
+
+        self._fit_cross_conditional_mean_outcome_nuisance(t, m, x, y)
+        self._fitted = True
+
+        if self.verbose:
+            print("Nuisance models fitted")
+
+        return self
+
+    @fitted
+    def estimate(self, t, m, x, y):
+        """Estimates causal effect on data
+
+        """
+
+        t, m, x, y = self._resize(t, m, x, y)
+
+        mu_0mx, mu_1mx, y0m0, y0m1, y1m0, y1m1 = self._estimate_cross_conditional_mean_outcome_nesting(
```

Comment:
avoid very long lines

Reply:
This is addressed in commit 6378d4318af2c23ec76999d758ad47475dd26d6e